### PR TITLE
roadmap

### PR DESCRIPTION
**Pull request by fabriziomoscon**

### CODE REVIEW

**Comment by fabriziomoscon** [comment]
@jdegger could you please review this? And suggest if I missed something.
I want to add this section to save time explaining the future plans in various issues.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,55 @@
...
+
+- Android 5.0.2
+- iOS 5.2.0
+
+It contains breaking changes from `react-native-twilio-programmable-voice` v4, and it will be released as v5.
+
+You can install it with:
+```bash
+yarn add https://github.com/hoxfon/react-native-twilio-programmable-voice#feat/twilio-android-sdk-5
+```
```
Comment (jdegger) [inline README.md:27]:
```suggestion
```bash
# Yarn
yarn add https://github.com/hoxfon/react-native-twilio-programmable-voice#feat/twilio-android-sdk-5

# NPM
npm install git+https://github.com/hoxfon/react-native-twilio-programmable-voice#feat/twilio-android-sdk-5
```
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,55 @@
...
+
+[iOS changelog](https://www.twilio.com/docs/voice/voip-sdk/ios/changelog)
+[Android changelog](https://www.twilio.com/docs/voice/voip-sdk/android/3x-changelog)
+
+My plan is to use the following links as a reference, and follow Twilio commit by commit.
+
+- https://github.com/twilio/voice-quickstart-android
+- https://github.com/twilio/voice-quickstart-ios
+
+_If you want to contribute please consider helping on this project._
```
Comment (jdegger) [inline README.md:39]:
```suggestion
_If you want to contribute please consider helping on this project. [Click here for more information](https://github.com/hoxfon/react-native-twilio-programmable-voice/issues/158)._
```

**Review by jdegger — CHANGES_REQUESTED** [review]
Two minor clarifications

**Comment by fabriziomoscon** [comment]
@jdegger I committed your changes. Could you approve?

**Review by jdegger — APPROVED** [review]

**Comment by jdegger** [comment]
@fabriziomoscon 👍

**Pull request merged**